tests/test_repoclosure.py

```diff
@@ -50,7 +50,7 @@ def test_check_option(self):
                 support.command_run(self.cmd, args)
 
             self.assertEqual(context.exception.value,
-                             "Repoclosure ended with unresolved dependencies.")
+                             "Repoclosure ended with unresolved dependencies (1) across 1 packages.")
 
             expected_out = ["package: foo-4-6.noarch from @commandline",
                             "  unresolved deps:",
```

Comment:
The tests are still failing, you have to also add the `(1)` here.
`"  unresolved deps (1):",`